binding/SkiaSharp/SKCanvas.cs

```diff
@@ -68,6 +68,16 @@ public int SaveLayer (SKPaint paint)
 			return SkiaApi.sk_canvas_save_layer (Handle, null, paint == null ? IntPtr.Zero : paint.Handle);
 		}
 
+		public int SaveLayer (SKRect bounds, SKPaint paint, SKImageFilter backdrop, SKSaveLayerFlags saveLayerFlags = SKSaveLayerFlags.None)
```

Comment:
Would it be worth creating a rec struct now instead of adding overloads? The next skia update will have a filters list and a color space, so reducing the new things to a new property may be nicer.

```cs
public struct SKCanvasSaveLayerRec
{
    public SKRect? Bounds { get; set; }
    public SKPaint? Paint  { get; set; }
    public SKImageFilter? Backdrop { get; set; }
    public SKCanvasSaveLayerFlags { get; set; }
}
```

Then in the save layer we can "unpack" this struct and pass it as the single C function. This way we don't need to create yet another overload to the save layer world just to add colorspace.